### PR TITLE
refactor: Changed Box Buttons component

### PR DESCRIPTION
**Pull request by jorgerivero3**
This is my first open source contribution. Right now I tried to only focus on BoxButton. Let me know if there are other locations this need to be placed in.

### CODE REVIEW

File: App/Screens/Home/SelectFrequency/SelectFrequency.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ import React, { useContext, useRef, useState } from 'react';
 import { ScrollView, ScrollViewProps, StyleSheet } from 'react-native';
 import { Frequency, FrequencyContext } from '@shootismoke/ui';
 
-import { BoxButton } from '../../../components';
+import { BoxButton } from '@shootismoke/ui/lib';
```
Comment (amaurymartiny) [inline App/Screens/Home/SelectFrequency/SelectFrequency.tsx:21]: Could put do `import { BoxButton, Frequency, FrequencyContext } from '@shootismoke/ui';` instead?

File: app.example.json [hunk 2]
```diff
@@ -13,7 +13,8 @@
 		},
 		"assetBundlePatterns": [
 			"assets/**/*",
-			"./node_modules/@shootismoke/ui/assets/**/*"
+			"./node_modules/@shootismoke/ui/assets/**/*",
+			"./node_modules/@shootismoke/ui/lib/**/*"
```
Comment (amaurymartiny) [inline app.example.json:17]: No, this line is incorrect, please revert 👍

**Comment by jorgerivero3** [comment]
The new UI makes the box buttons smaller
new component
old component

**Comment by jorgerivero3** [comment]
Fixed all of your comments. Thank you 👍

**Pull request merged**